packages/plugin-search/src/search.js

```diff
@@ -0,0 +1,115 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {pick, isArray} from 'lodash';
+import {SparkPlugin} from '@ciscospark/spark-core';
+
+const Search = SparkPlugin.extend({
+  namespace: `Search`,
+
+  people(options) {
+    options = options || {};
+
+    if (!options.query && !options.queryString) {
+      return Promise.reject(new Error(`'options.query' is required`));
+    }
+
+    const body = pick(options, `size`, `includePeople`, `includeRooms`, `includeRobots`);
```

Comment:
In order to avoid needing to make SDK changes when we add support for a new parameter, I've been rearranging thing so that the options passed to the new sdk methods are generally just forwarded to the api. for the convo list and get methods, for example, all but a few options are passed directly to the query string.